### ISSUE TITLE
Standard folders are nested

### ISSUE THREAD

**Issue by mmats14** [deohs/folders#1]
I tried using this package today to create standard folders in a new project (using code chunks in the readme). 
```
pacman::p_load(here)
pacman::p_load_gh("deohs/folders")

# Get the list of standard folders and create any folders which are missing.
folders <- get_folders()
result <- create_folders(folders)
```

The folders are nested such that I have: data > doc > figures > results, rather than each of the folders within the project root directory. I've used it in the past with no issues ~5 months ago.

**Comment by brianhigh** [comment]
Thanks. Just saw this. Sorry didn't see sooner. I'll take a look.

**Comment by brianhigh** [comment]
This appears to have been fixed with commit fbe35b7 which also addressed issue #2. Version is now 0.0.3.
